### Ladda Pythonpaket

In [1]:
import numpy as np
import sympy as sp
from numpy import linalg as LA
import math as mt
#from matplotlib.image import imread
import matplotlib.pyplot as plt
from matplotlib import rcParams # for changing default values
import scipy.io as sio
import scipy.optimize
from scipy.optimize import minimize
from scipy.optimize import fsolve
from scipy.integrate import odeint
import os


## U1

<img src="images/statik/u1.png" style="width:30%;">



a) Frilägg och inför stödreaktioner (Draw free-body diagram)

b) Bestäm stödreaktionerna (Determine reactions)

<details>
<summary><strong>Ledtråd a)</strong> (click to show)</summary>


<img src="images/statik/U1sol.png" style="width:20%;">

</details>





In [2]:
#lösning b)
VA,HA,VB,F,M,a=sp.symbols('VA,HA,VB,F,M,a')

krafter_uppat=VA+VB-F*np.sin(np.pi/4) #Jämvikt vertikalt
krafter_sidled=HA+F*np.cos(np.pi/4) #Jämvikt sidled
Moment_A=M-F*np.sin(np.pi/4)*2*a-VB*3*a #Momentjämvikt kring A
losning=sp.solve([krafter_uppat,krafter_sidled,Moment_A],[VA,HA,VB])

print("HA=",losning[HA])
print("VA=",sp.expand(losning[VA]))
print("VB=",sp.expand(losning[VB]))

HA= -0.707106781186548*F
VA= 1.17851130197758*F - 0.333333333333333*M/a
VB= -0.471404520791033*F + 0.333333333333333*M/a


<img src="images/statik/BL1.3.png" style="width:30%;">

<img src="images/statik/BL1.6.png" style="width:30%;">



<details>
<summary><strong>Ledtråd</strong> (click to show)</summary>

Friläggning:

<img src="images/statik/BL1.6sol.png" style="width:10%;">

</details>







<img src="images/statik/BL1.20.png" style="width:30%;">


<details>
<summary><strong>Lösning</strong> (click to show)</summary>


<img src="images/statik/BL1.20losning.png" style="width:10%;">

</details>

<img src="images/statik/BL1.28.png" style="width:30%;">


<details>
<summary><strong>Lösning</strong> (click to show)</summary>

Friläggning:

<img src="images/statik/BL1.28sol.png" style="width:30%;">

</details>






## Hemtal

<img src="images/statik/BL1.1and1.2.png" style="width:30%;">

<details>
<summary><strong>Ledtråd 1.1 friläggning</strong> (click to show)</summary>

<img src="images/statik/BL1.1frilaggning.png" style="width:30%;">

</details>

<details>
<summary><strong>Ledtråd 1.1 jämvikt</strong> (click to show)</summary>


img src="images/statik/BL1.1jamvikt.png" style="width:30%;">

</details>

<details>
<summary><strong>Ledtråd 1.2</strong> (click to show)</summary>

Jämviktsekvationerna ändras inte.

</details>

<img src="images/statik/BL1.4.png" style="width:30%;">

<details>
<summary><strong>Ledtråd 1.4 friläggning</strong> (click to show)</summary>

<img src="images/statik/BL1.4frilaggning.png" style="width:20%;">

</details>



<img src="images/statik/BL1.15.png" style="width:30%;">

<details>
<summary><strong>Ledtråd 1.15 (nästan komplett lösning)</strong> (click to show)</summary>

<img src="images/statik/BL1.15losning.png" style="width:30%;">

</details>


<img src="images/statik/BL1.21.png" style="width:30%;">

<details>
<summary><strong>Ledtråd</strong> (click to show)</summary>

<img src="images/statik/BL1.21sol.png" style="width:20%;">

</details>



<img src="images/statik/BL1.22.png" style="width:30%;">

<details>
<summary><strong>Ledtråd friläggning</strong> (click to show)</summary>

<img src="images/statik/BL1.22frilaggning.png" style="width:20%;">

</details>

<details>
<summary><strong>Ledtråd geometri</strong> (click to show)</summary>

Friläggning:

<img src="images/statik/BL1.22geometri.png" style="width:20%;">

</details>

<img src="images/statik/BL1.26.png" style="width:30%;">

<details>
<summary><strong>Ledtråd</strong> (click to show)</summary>

<img src="images/statik/BL1.26ledtrad.png" style="width:20%;">
</details>

<img src="images/statik/BL1.30.png" style="width:30%;">

<details>
<summary><strong>Ledtråd</strong> (click to show)</summary>

<img src="images/statik/BL1.30ledtrad.png" style="width:30%;">
</details>